src/xdocs/config_filters.xml

```diff
@@ -985,6 +985,27 @@ public class InputTest {
       </subsection>
     </section>
 
+    <section name="SupperssionXpathSingleFilter">
+      <subsection name="Description" id="SuppressionXpathSingleFilter">
+
```

Comment:
See http://checkstyle.sourceforge.net/config_filters.html#SuppressionXpathFilter and how it is documented. Since all this filter is is a wrapper of the other, documentation may be similar. Big differences would be that this filter needs to describe itself as a single suppression that is written in the configuration, instead of pointing to an external file that supports multiple suppressions.

Reply:
How should I write the "since time" of this filter? Should I write the latest version 8.1? Or just follow the "SuppressionXpathFilter" with "since 3.5".